scripts/tests/test_zkai_paper_claim_pack_gate.py

```
@@ -0,0 +1,174 @@
+    def test_referenced_evidence_paths_exist(self):
+        paths = [gate.ROOT / ref["path"] for ref in self.payload["evidence_refs"]]
+        self.assertGreaterEqual(len(paths), 8)
+        for path in paths:
+            self.assertTrue(path.is_file(), path)
+
+    def test_all_declared_mutations_reject(self):
+        expected = set(gate.EXPECTED_MUTATION_NAMES)
+        actual = {name for name, _ in gate.mutation_cases(self.payload)}
+        self.assertEqual(actual, expected)
+        for name, mutated in gate.mutation_cases(self.payload):
+            with self.assertRaises(gate.ClaimPackGateError, msg=name):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

(PT027)